styleframe/style_frame.py

```diff
@@ -249,7 +252,10 @@ def read_excel_as_template(cls, path, df, use_df_boundaries=False, **kwargs):
 
         if use_df_boundaries:
             sf.data_df = sf.data_df.iloc[:num_of_rows, :num_of_cols]
-
+            sf._rows_height = {k: v for i, (k, v) in enumerate(sf._rows_height.items())
```

Comment:
1. sf._row_height is now OrderedDict not regular dict (you use dict comprehension not OrderedDict)
2. `sf._row_height = OrderDict(sf._rows_height.items()[:num_of_rows])`

Reply:
We should even consider to implement setter to `_row_height` and `_column_width` that validate this or convert dict to `OrderedDict`